tests/test_fixity.py

```diff
@@ -551,3 +470,72 @@ def test_main_handles_exceptions_if_scanall_fails(_get, monkeypatch):
             "Successfully scanned 2 AIPs",
         ],
     )
+
+
+@mock.patch("requests.get")
+def test_scanall_if_sort_argument_is_passed(_get, environment, mock_check_fixity):
+    aip1_uuid = str(uuid.uuid4())
+    aip2_uuid = str(uuid.uuid4())
+    _get.side_effect = [
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "meta": {"next": None},
+                    "objects": [
+                        {
+                            "package_type": "AIP",
+                            "status": "UPLOADED",
+                            "uuid": aip1_uuid,
+                        },
+                        {
+                            "package_type": "AIP",
+                            "status": "UPLOADED",
+                            "uuid": aip2_uuid,
+                        },
+                    ],
+                },
+            },
+            spec=requests.Response,
+        ),
+        *mock_check_fixity,
+        *mock_check_fixity,
+    ]
+
+    stream = io.StringIO()
+
+    response = fixity.main(["scanall", "--sort"], stream=stream)
+
+    assert response == 0
+
+    _assert_stream_content_matches(
+        stream,
+        [
+            f"Fixity scan succeeded for AIP: {aip1_uuid}",
+            f"Fixity scan succeeded for AIP: {aip2_uuid}",
+            "Successfully scanned 2 AIPs",
```

Comment:
Could you please add a couple of errors to this test case? Something along the lines of:

* Fixity scan of aip1 succeeds
* Fixity scan of aip2 fails
* Fixity scan of aip3 succeeds
* Fixity scan of aip4 fails

And check the output should list them as:

* Fixity scan of aip2 failed
* Fixity scan of aip4 failed
* Fixity scan of aip1 succeeded
* Fixity scan of aip3 succeeded
* Successfully scanned 4 AIPs